notebooks/algorithms/structure/Renumber.ipynb

```diff
@@ -57,7 +57,11 @@
     "import pandas as pd\n",
     "import numpy as np\n",
     "import networkx as nx\n",
-    "from cugraph.structure import NumberMap"
+    "from cugraph.structure import NumberMap\n",
+    "\n",
+    "# Force this notebook to fail\n",
+    "# TODO: Remove this before merging\n",
+    "exit(1)"
```

Comment:
Should this have been removed?

Reply:
I put it in for testing. The thing I tested was successful, so I'll remove it.